templates/zerver/help/search-for-messages.md

```diff
@@ -3,69 +3,125 @@
-* `is:mentioned`: See [mentions](/help/mention-a-user-or-group).
-* `is:starred`: See [starred messages](/help/star-a-message).
+* `pm-with:Bo Lin`: Search 1-on-1 private messages between you and Bo.
+* `pm-with:Bo Lin, Elena García`: Search group private messages
+  between you, Bo, and Elena.
+* `group-pm-with:Bo Lin`: Search all group private message
+  conversations that include you and Bo, as well as any other users.
+* `streams:public`: Search the history of all [public
+  streams](/help/change-the-privacy-of-a-stream) in the organization, including
+  streams you are not subscribed to; see details
+  [below](#searching-shared-history).
+
+### Search by sender
+
+* `sender:Elena García`: Search messages sent by Elena.
+* `sender:me`: Search messages you've sent.
+
+!!! tip ""
+
+    You can also access all messages someone has sent [via their **user
+    card**](/help/view-messages-sent-by-a-user).
+
+### Search for attachments or links
+
+* `has:link`: Search messages that contain URLs.
+* `has:attachment`: Search messages that contain an [uploaded
+  file](/help/share-and-upload-files).
+* `has:image`: Search messages that contain an uploaded image or link to an image.
+
+!!! tip ""
+
+    You can also [view](/help/manage-your-uploaded-files) all the files you have uploaded
+    or [browse](/help/view-and-browse-images) all the images in the current view.
+
+### Search your important messages
+
+* `is:alerted`: Search messages that contain your [alert
+  words](/help/pm-mention-alert-notifications#alert-words).
+* `is:mentioned`: Search messages where you were
+  [mentioned](/help/mention-a-user-or-group).
+* `is:starred`: Search your [starred messages](/help/star-a-message).
+
+### Search by message status
+
 * `is:resolved`: Search messages in [resolved topics](/help/resolve-a-topic).
 * `-is:resolved`: Search messages in [unresolved topics](/help/resolve-a-topic).
-* `is:unread`
-* `has:link`
-* `has:image`
-* `has:attachment`
-* `streams:public`: Search the history of all [public
-  streams](/help/change-the-privacy-of-a-stream) in the organization,
-  including streams you are not subscribed to.
+* `is:unread`: Search your unread messages.
 
-### Excluding messages
+### Search by message ID
+
+Each message in Zulip has a unique ID, which is used for [linking to a specific
+message](/help/link-to-a-message-or-conversation#link-to-zulip-from-anywhere).
+You can use the search bar to navigate to a message by its ID.
+
+* `near:12345`: Show messages around the message with ID `12345`.
+* `id:12345`: Show only message `12345`.
+* `stream:design near:1` Show the oldest messages in the **#design** stream.
 
-Zulip's search operators can be negated, to exclude messages matching
-the rule.  For example, `stream:design -is:resolved -has:image` will
-match all messages that don't contain an image and were sent to
-unresolved topics in the `#design` stream.
+### Negated filters
```

Comment:
Maybe we should consider "Exclude filters" here?  I think it's a more common term elsewhere and it would sound a bit less technical compared to "Negated filters".

Reply:
"Exclude" is the verb we use in the search suggestions dropdown menu.